src/benchmarklib/benchmark_runner.hpp

```diff
@@ -51,6 +51,11 @@ class BenchmarkRunner : public Noncopyable {
   // If the query execution should be validated, this stores a pointer to the used SQLite instance
   std::shared_ptr<SQLiteWrapper> sqlite_wrapper;
 
+  // Create a report in roughly the same format as google benchmarks do when run with --benchmark_format=json.
+  // This is idempotent, i.e., you can call it multiple times and the result will be updated. Be aware that this may
```

Comment:
That "be updated" confused me a bit, since you "just overwrite" the whole file, don't you?

Reply:
`and the result file will be overwritten`?

Reply:
👍